src/util/ResourceMapper.ts

```diff
@@ -0,0 +1,19 @@
+import { types } from 'mime-types';
+
+// This class could contain more URL/file mapping logic.
+export class ResourceMapper {
+  private readonly types: Record<string, any>;
+
+  public constructor(
+    overrideTypes = { acl: 'text/turtle', metadata: 'text/turtle' },
```

Comment:
We have a constant for `'text/turtle'` in `ContentTypes.ts`.

Reply:
I will actually solve this in #142